doc/source/User_guide/files.rst

```diff
@@ -0,0 +1,70 @@
+Load and manage files
+=====================
+
+PyAEDT supports different files formats to manage simulation and setup data.
+The most used are described in this chapter.
+
+
+.. grid:: 2
+
+   .. grid-item-card:: Project file
+            :link: pyaedt_file_data/project
+            :link-type: doc
+
+            The project configuration file allows to apply a set of variables, materials, setup, mesh and boundaries.
```

Comment:
```suggestion
            The project configuration file allows you to apply a set of variables, materials, setup, mesh, and boundaries.
```
We use the "Oxford" comma, which says to place a command before the last item in an "and" or "or" list. The phrase "allows to apply" isn't valid English. The "you" is necessary.